modules/connectionCheck.py

```diff
@@ -10,13 +11,20 @@ class ConnectionCheck(modules.Module):
     GRAPHS = {}
     for code, graph in [
         ('7HPJ', {(1, 2), (2, 3), (1, 3), (4, 6), (5, 6), (5, 7), (4, 7)}),
-        ('34XYZ', {(1, 4), (1, 2), (1, 6), (2, 3), (2, 4), (4, 7), (6, 7), (7, 8), (6, 8), (5, 6)}),
-        ('SLIM', {(1, 2), (1, 3), (1, 6), (2, 6), (3, 6), (3, 4), (4, 6), (5, 6), (4, 5), (4, 8), (4, 7), (5, 7), (7, 8)}),
-        ('15BRO', {(1, 7), (1, 2), (2, 7), (6, 7), (5, 6), (5, 7), (4, 8), (3, 8), (3, 4)}),
-        ('8CAKE', {(1, 3), (2, 4), (3, 6), (1, 2), (3, 4), (3, 7), (4, 7), (5, 7), (7, 8), (4, 5), (3, 8), (5, 8), (1, 8), (1, 6), (2, 6), (4, 6)}),
-        ('9QVN', {(1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7), (7, 8), (1, 8), (2, 7), (3, 6), (2, 6), (3, 7), (1, 4), (5, 8)}),
-        ('20DGT', {(1, 3), (3, 5), (5, 7), (2, 7), (2, 4), (4, 6), (6, 8), (3, 7), (4, 7), (4, 8), (1, 2), (5, 6)}),
-        ('6WUF', {(2, 3), (3, 8), (3, 5), (3, 6), (4, 5), (4, 7), (4, 8), (5, 7), (5, 6), (7, 8), (2, 8), (6, 7), (1, 6), (1, 7), (1, 2), (2, 7)}),
+        ('34XYZ', {(1, 4), (1, 2), (1, 6), (2, 3),
+                   (2, 4), (4, 7), (6, 7), (7, 8), (6, 8), (5, 6)}),
+        ('SLIM', {(1, 2), (1, 3), (1, 6), (2, 6), (3, 6), (3, 4),
+                  (4, 6), (5, 6), (4, 5), (4, 8), (4, 7), (5, 7), (7, 8)}),
+        ('15BRO', {(1, 7), (1, 2), (2, 7), (6, 7),
+                   (5, 6), (5, 7), (4, 8), (3, 8), (3, 4)}),
+        ('8CAKE', {(1, 3), (2, 4), (3, 6), (1, 2), (3, 4), (3, 7), (4, 7),
+                   (5, 7), (7, 8), (4, 5), (3, 8), (5, 8), (1, 8), (1, 6), (2, 6), (4, 6)}),
+        ('9QVN', {(1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7), (7,
+                                                                   8), (1, 8), (2, 7), (3, 6), (2, 6), (3, 7), (1, 4), (5, 8)}),
+        ('20DGT', {(1, 3), (3, 5), (5, 7), (2, 7), (2, 4),
+                   (4, 6), (6, 8), (3, 7), (4, 7), (4, 8), (1, 2), (5, 6)}),
+        ('6WUF', {(2, 3), (3, 8), (3, 5), (3, 6), (4, 5), (4, 7), (4, 8), (5,
+                                                                           7), (5, 6), (7, 8), (2, 8), (6, 7), (1, 6), (1, 7), (1, 2), (2, 7)}),
```

Comment:
The code style is a guideline, not a bible. Automatic code formatters are a bad idea because they can't exercise their judgement. This is the most egregious example in the diff so far, but there are other instances of the code becoming uglier for the sake of checking an arbitrary checkbox.

Automatic reformatting should be reserved to the most clear-cut, hand-picked rules. I won't be merging this commit as is, and I would ask you to change formatting only in the vicinity of actually meaningful changes.

Reply:
I see what you mean 🤦‍♂